data-table-cell.html

```diff
@@ -58,6 +80,15 @@
                   '_orderChanged(order)',
                   '_widthChanged(width)'],
 
+      listeners: {
+        'dragstart': '_handleDragStart',
```

Comment:
As the functionality for reordering is isolated, I'd like to have all the logic encapsulated in a separate behavior. `ColumnReorderingBehavior` maybe?